src/st_stuff.c

```diff
@@ -376,14 +376,7 @@ void ST_refreshBackground(boolean force)
         return;
     }
 
-    if (SHORT(sbar->width) > video.unscaledw && SHORT(sbar->leftoffset) == 0)
-    {
-        st_x = ST_X + (SCREENWIDTH - SHORT(sbar->width)) / 2;
-    }
-    else
-    {
-        st_x = ST_X;
-    }
+    st_x = ST_X + (SCREENWIDTH - SHORT(sbar->width)) / 2 + SHORT(sbar->leftoffset);
```

Comment:
Alternatively, we could force `sbar->leftoffset = 0` after loading that lump, just as we do in `V_DrawPatchFullScreen()`.

Reply:
Not sure what's best. Should I keep it as it is for now?

Reply:
Yes, okay.